frontend/src/views/WheresMyGene/components/RightSideBar/style.ts

```diff
@@ -1,10 +1,9 @@
 import styled from "@emotion/styled";
-import {
-  SideBarPositioner as RawSideBarPositioner,
-  SIDEBAR_BOX_SHADOW_COLOR,
-} from "src/components/common/SideBar/style";
+import { SideBarPositioner as RawSideBarPositioner } from "src/components/common/SideBar/style";
 import { HEADER_HEIGHT_PX } from "src/components/Header/style";
 
+export const SIDEBAR_BOX_SHADOW_COLOR = "rgba(16, 22, 26, 0.15)";
```

Comment:
Refactored from Sidebar styles - this box shadow colour is localised to RightSideBar component. See [slack discussion](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1684357377546829?thread_ts=1684236150.259999&cid=C02CF7MQPGV).